flink-ml-core/src/main/java/org/apache/flink/ml/common/window/Window.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.common.window;
+
+/**
+ * A {@link Window} determines how input data stream would be sliced into batches and fed into a
+ * Flink ML Stage.
+ */
+public interface Window {}
```

Comment:
Since this class represents window strategy, not concrete window instances, would it be better to rename this class as `Windows`?

Note that Flink names similar concepts as `TumblingEventTimeWindows`. Beam names similar concepts as `SlidingWindows` [1]. 

[1] https://beam.apache.org/documentation/programming-guide/#using-sliding-time-windows